regression/regression_param.sh

```diff
@@ -103,9 +111,12 @@ case $regtest in
         elif [[ "$machine" = "Jet" ]]; then
            topts[1]="0:15:00" ; popts[1]="5/4/"  ; ropts[1]="/1"
            topts[2]="0:15:00" ; popts[2]="10/4/"  ; ropts[2]="/1"
-        elif [[ "$machine" = "Gaea" ]]; then
-           topts[1]="0:15:00" ; popts[1]="64/1/"  ; ropts[1]="/1"
-           topts[2]="0:15:00" ; popts[2]="128/2/"  ; ropts[2]="/1"
+        elif [[ "$machine" = "gaeac5" ]]; then
+           topts[1]="0:15:00" ; popts[1]="40/3/"  ; ropts[1]="/1"
+           topts[2]="0:15:00" ; popts[2]="40/5/"  ; ropts[2]="/1"
+        elif [[ "$machine" = "gaeac6" ]]; then
+           topts[1]="0:60:00" ; popts[1]="40/3/"  ; ropts[1]="/1"
```

Comment:
The C6 wall clock limit request on this and the following line can be reduced to 15 minutes.  A check of recent `rrfs_3denvar_rdasens` tests on C5 and C6 return `gsi.x` wall times ranging from 77 to 126 seconds.

Reply:
Hi @RussTreadon-NOAA. I returned the wall clock for rrfs to 15 mins for C6.  I also moved the placement of `export FI_VERBS_PREFER_XRC=0` from `ush/sub_gaeac5` to `regression/regression_var.sh`. Let me know if that is not okay.  I reran the C5 regression tests successfully after making this change.  And I am getting similar behavior that you are.  The jobs are consistently completing successfully but with a bit of variability in run time.
When is WCOSS returned to the devs?

Reply:
I moved FI_VERBS_PREFER_XRC and re committed the change.